yarn.lock

```diff
@@ -7174,16 +7174,22 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@testing-library/react-native@npm:^7.1.0":
-  version: 7.2.0
-  resolution: "@testing-library/react-native@npm:7.2.0"
+"@testing-library/react-native@npm:^12.5.2":
```

Comment:
Nice bump 👀

Reply:
I had to bump this version, otherwise `screen` was not defined. Lmk if you prefer lower version 😝